src/main/java/cz/czechitas/automation/LoginAction.java

```diff
@@ -20,33 +20,33 @@ final class LoginAction {
     }
 
     void clickLoginMenuLink() {
-        var loginButton = elementFinder.findByXPath("/html/body/div/header/nav/div/div[2]/a");
+        var loginButton = elementFinder.findByCssSelector(".navbar-right .nav-item");
         loginButton.click();
     }
 
     void insertEmail(String email) {
         Objects.requireNonNull(email);
 
-        var emailInputBox = elementFinder.findByXPath("//*[@id=\"email\"]");
+        var emailInputBox = elementFinder.findByXPath("//*[@id='email']");
         emailInputBox.sendKeys(email);
     }
 
     void insertPassword(String password) {
         Objects.requireNonNull(password);
 
-        var passwordInputBox = elementFinder.findByXPath("//*[@id=\"password\"]");
+        var passwordInputBox = elementFinder.findByXPath("//*[@id='password']");
         passwordInputBox.sendKeys(password);
     }
 
     void clickLoginButton() {
-        var loginButton = elementFinder.findByXPath("/html/body/div/div/div/div/div/div/form/div[3]/div/button");
+        var loginButton = elementFinder.findByXPath("//button[@type='submit']");
         loginButton.click();
     }
 
     void logout() {
-        var signedInUserElement = elementFinder.findByXPath("//*[@id=\"navbarSupportedContent\"]/div[2]/div/a");
+        var signedInUserElement = elementFinder.findByXPath("//*[@id='navbarSupportedContent']/div[2]/div/a");
```

Comment:
I would not rely that much on the order 🙂 
```suggestion
        var signedInUserElement = elementFinder.findByXPath("//*[@id='navbarSupportedContent']//a[@class='dropdown-toggle']");
```

Reply:
Fixed